api/v1alpha1/oadp_types.go

```diff
@@ -115,13 +115,13 @@ type PodConfig struct {
 	// labels to add to pods
 	// +optional
 	Labels map[string]string `json:"labels,omitempty"`
-	// nodeSelector defines the nodeSelector to be supplied to podSpec
+	// nodeSelector defines the nodeSelector to be supplied to NodeAgent podSpec
 	// +optional
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
-	// tolerations defines the list of tolerations to be applied
+	// tolerations defines the list of tolerations to be applied to NodeAgent daemonset
 	// +optional
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
-	// resourceAllocations defines the CPU and Memory resource allocations for the Pod
+	// resourceAllocations defines the CPU and Memory resource allocations for the NodeAgent Pod
```

Comment:
NIT: same change as suggested for `tolerations` and `nodeSelector`

Reply:
Fixed.